src/components/calcite-date-picker-day/calcite-date-picker-day.scss

```diff
@@ -73,19 +74,18 @@
 }
```

Comment:
Address these height / width props?

Reply:
Not sure what you mean, they are in s, m, l style props.

Reply:
@jrowlingson they are odd numbers off scale so that day can be perfectly centered I believe?

Reply:
yea, they are not aligned if I change value to scale